common/decklist.cpp

```diff
@@ -537,20 +585,25 @@ bool DeckList::loadFromStream_Plain(QTextStream &in)
         // Replace only if the ampersand is preceded by a non-capital letter,
         // as would happen with acronyms. So 'Fire & Ice' is replaced but not
         // 'R&D' or 'R & D'.
-        //
-        // Qt regexes don't support lookbehind so we capture and replace
-        // instead.
+        // Qt regexes don't support lookbehind so we capture and replace instead.
         rx.setPattern("([^A-Z])\\s*&\\s*");
-        if (rx.indexIn(cardName) != -1) {
+        if (rx.indexIn(cardName) != -1)
             cardName.replace(rx, QString("%1 // ").arg(rx.cap(1)));
-        }
 
-        // Look for the correct card zone
-        QString zoneName = getCardZoneFromName(cardName, isSideboard ? DECK_ZONE_SIDE: DECK_ZONE_MAIN);
+        // We need to get the name of the card from the database,
+        // but we can't do that until we sanitize the name
```

Comment:
when does sanitization happening? I cant see a usage of it in `getCompleteCardName` or its stack.